tests/MongoDB.EntityFrameworkCore.UnitTests/Infrastructure/MongoOptionsExtensionTest.cs

```diff
@@ -46,18 +45,47 @@ public static void Can_set_mongo_client_and_database_name(string databaseName)
         Assert.Equal(databaseName, options.DatabaseName);
     }
 
+    [Theory]
+    [InlineData("SomeDatabase")]
+    public static void Can_set_mongo_client_settings_and_database_name(string databaseName)
```

Comment:
Why is this a theory when it only has one data value?

Reply:
I think this started as a theory with multiple values and then stopped having them. I can create a separate PR to turn them back if needed.